docs/use/multi-namespaces.md

```diff
@@ -30,41 +30,35 @@ To set up the namespaces that Percona Everest will manage and where you can depl
 
 The following holds true for multiple namespaces:
 
-- You can configure multiple namespaces in Percona Everest. 
+- You can [configure multiple namespaces](../administer/manage_namespaces.md#add-new-namespaces) in Percona Everest using the `everestctl namespaces add [NAMESPACE]` command.
 
-- You can install different operators in various namespaces by running the install command multiple times.
+- You can [install different operators](../administer/manage_namespaces.md#update-namespaces) in various namespaces using the `everestctl namespaces update [NAMESPACE]` command.
 
-    See the following example for more details:
-    
-    ??? info "Example: Installing various operators in different namespaces"
-            
-        To install various operators in different namespaces, such as MySQL and MongoDB operator in namespaces dev and testing, and PostgreSQL operator in namespaces production and staging, run the following commands:
-            {.power-number}
 
-        1. `everestctl install`
+??? example "Example: Configuring multiple namesapces and installing various operators within those namespaces"            
+    To install various operators in different namespaces, such as MongoDB and MySQL operator in namespace production, and PostgreSQL operator in namespace development, run the following commands:
+    {.power-number}
 
+    1. `everestctl namespaces add development`
                 
-            ??? example "Expected output"
+        ??? example "Expected output"
+            ```sh
+            ? Which operators do you want to install? PostgreSQL
+            ✓ Installing namespace 'development'
+            ```
 
-                ```sh
-                ? Namespaces managed by Everest [comma separated] dev,testing
-                ? What operators do you want to install? MySQL, MongoDB
-                ```
+    2. `everestctl namespaces add production`.
 
-        2. Rerun the `everestctl install` command to install the PostgreSQL operator in the production and staging namespaces.
+        ??? example "Expected output"
+            ```sh
+            ? Which operators do you want to install? MySQL, MongoDB
+            ✓ Installing namespace 'production'
+            ```
```

Comment:
Maybe we can switch the order of these 2 examples because in the description above we first mention production and then development